.travis.yml

```diff
@@ -3,18 +3,18 @@
   sudo: required
   language: go
   dist: trusty
-  osx_image: xcode9
+  osx_image: xcode10.1
   os:
     - linux
     - osx
   notifications:
     email: false
   go:
-    - 1.8.1
+    - 1.12.1
   before_install:
     # work-around for issue https://github.com/travis-ci/travis-ci/issues/6307
     # might not be necessary in the future
-    - command curl -sSL https://rvm.io/mpapis.asc | gpg --import -
+    - command gpg --verbose --keyserver hkp://fks.pgpkeys.eu --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
```

Comment:
did we check if the workaround is still needed? I see https://github.com/travis-ci/travis-ci/issues/6307 was closed, so perhaps updating is no longer needed?

Reply:
Indeed, this workaround is no more necessary.